From a5c2a2bf02e3dfc16054d392d13cd246cb06b0a4 Mon Sep 17 00:00:00 2001
From: Ian Abbott <abbotti@mev.co.uk>
Date: Mon, 6 Apr 2009 15:10:30 -0400
Subject: Staging: comedi: 'pcmcia_parse_tuple()' now has two arguments, not three.

From: Ian Abbott <abbotti@mev.co.uk>

This fixes the build error in the cv_das16_cs driver

From: Ian Abbott <abbotti@mev.co.uk>
Signed-off-by: Frank Mori Hess <fmhess@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/cb_das16_cs.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/comedi/drivers/cb_das16_cs.c
+++ b/drivers/staging/comedi/drivers/cb_das16_cs.c
@@ -773,7 +773,7 @@ static void das16cs_pcmcia_config(struct
 	if ((last_ret = pcmcia_get_tuple_data(link, &tuple)) != 0)
 		goto cs_failed;
 	last_fn = ParseTuple;
-	if ((last_ret = pcmcia_parse_tuple(link, &tuple, &parse)) != 0)
+	if ((last_ret = pcmcia_parse_tuple(&tuple, &parse)) != 0)
 		goto cs_failed;
 	link->conf.ConfigBase = parse.config.base;
 	link->conf.Present = parse.config.rmask[0];
@@ -798,7 +798,7 @@ static void das16cs_pcmcia_config(struct
 		cistpl_cftable_entry_t *cfg = &(parse.cftable_entry);
 		if (pcmcia_get_tuple_data(link, &tuple))
 			goto next_entry;
-		if (pcmcia_parse_tuple(link, &tuple, &parse))
+		if (pcmcia_parse_tuple(&tuple, &parse))
 			goto next_entry;
 
 		if (cfg->flags & CISTPL_CFTABLE_DEFAULT)
